datafusion/sqllogictest/test_files/cse.slt

```diff
@@ -179,20 +179,20 @@ physical_plan
 # Surely only once but also conditionally evaluated expressions
 query TT
 EXPLAIN SELECT
-    (a = 1 OR random() = 0) AND a = 1 AS c1,
-    (a = 2 AND random() = 0) OR a = 2 AS c2,
+    (a = 1 OR random() = 0) AND a = 2 AS c1,
```

Comment:
These changes are neccessary to avoid us simplifiying these conditions and therefore changing what is gets tested here.